README.md

```diff
@@ -1,8 +1,38 @@
-#Yazan's notes
 
-Install requirements
-Make sure you have a .env file created with gmail information
 
+# Setup Instructions
+
+
+
+
+* Create two env files
+
+    login1.env
+
+        GMAIL1 = ""
```

Comment:
Putting the logins in the Readme file removes the point of having the login info in the env files.